src/lib/validationRules/decentClientName/decentClientName.ts

```diff
@@ -18,7 +18,31 @@
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
-import { ValidationContext, ValidationRule } from "../../types";
+import {
+  ResultDescription,
+  ValidationContext,
+  ValidationRule,
+} from "../../types";
+
+const resultDescriptions: Record<string, ResultDescription> = {
+  missingClientName: {
+    status: "warning",
+    title: "No Client Name present",
+    description:
+      "The document has no Client Name set. It should be set for authentication providers to display the name to the end-user.",
+  },
+  invalidClientName: {
+    status: "error",
+    title: "Invalid Client Name",
+    description: "The Client Name is not of type string",
+  },
+  longClientName: {
+    status: "warning",
+    title: "Long Client Name",
+    description:
+      "The length of the Client Name is longer than 50 characters. This might cause presentation problems.",
+  },
+};
```

Comment:
Maybe I'm overseeing an option but since I'm using the string keys as reference in the `check` methods to generate the results, that seemed to me the most straight forward way of going ahead.

Reply:
Oh yeah my bad I missed that ><